agents/tools/media/audio_analysis.py

```
@@ -0,0 +1,139 @@
+    raw_tags = inputs.get("context_tags", [])
+    if isinstance(raw_tags, list):
+        context_tags = [str(tag) for tag in raw_tags if str(tag).strip()]
+    elif isinstance(raw_tags, str):
+        context_tags = [t.strip() for t in raw_tags.split(",") if t.strip()]
+
+    metrics: Dict[str, Any]
+    analysis_source = "librosa"
+    if librosa is not None:
+        try:
+            metrics = _analyze_with_librosa(path)
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)